app/src/components/cards/collectionCard.tsx

```diff
@@ -6,56 +6,67 @@ import {
   Text,
   CardContent,
   Tooltip,
-  Link,
+  StatusBadge,
 } from "@nypl/design-system-react-components";
 import styles from "./Card.module.css";
 import { headerBreakpoints } from "../../utils/breakpoints";
 import { CollectionCardDataType } from "../../types/CollectionCardDataType";
-
 interface CollectionCardProps {
   slug: string;
   id: number;
   isLargerThanLargeTablet: boolean;
+  isLargerThanDesktop: boolean;
   collection: CollectionCardDataType;
 }
 
 const CollectionCard = ({
   slug,
   id,
   isLargerThanLargeTablet,
+  isLargerThanDesktop,
   collection,
 }: CollectionCardProps) => {
-  return (
+  const truncatedTitle = collection.title.length > 80; // Pretty much random
+  const card = (
     <Card
-      sx={{ display: "grid" }}
       id={`card-${slug}-${id}`}
       mainActionLink={collection.url}
-      imageProps={{
-        alt: "",
-        id: `image-${slug}-${id}`,
-        isLazy: true,
-        aspectRatio: "twoByOne",
-        src: collection.imageURL,
-      }}
+      imageProps={
+        collection.imageID
+          ? {
+              alt: "",
+              id: `image-${slug}-${id}`,
+              isLazy: true,
+              aspectRatio: "twoByOne",
+              src: collection.imageURL,
+            }
+          : {
+              alt: "",
+              id: `no-image-${id}`,
+              isLazy: true,
+              aspectRatio: "twoByOne",
+              src: "/noImage.png",
+            }
+      }
     >
+      <CardContent>
+        {collection.containsOnSiteMaterials && (
```

Comment:
If it's not a layout problem, include `CardContent` in this conditional render logic. Small thing but one less div on the page.

Reply:
Layout problem for the `Card` :(